src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,22 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
```

Comment:
## Where I put these goals in the makefile
I made this section. Wasn't sure where else to put it.